src/Integration.Vsix/AsmRef_Integration.Vsix_Baseline_WithoutStrongNames.txt

```diff
@@ -23,35 +23,37 @@
 Include patterns:
 - 'Sonar*'
 Exclude patterns:
-# Number of matches: 20
+# Number of matches: 21
 
 ---
-Assembly: 'SonarAnalyzer.CFG, Version=9.30.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+Assembly: 'SonarAnalyzer.CFG, Version=9.32.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
 Relative path: 'SonarAnalyzer.CFG.dll'
 
 Referenced assemblies:
 - 'Microsoft.CodeAnalysis, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'Microsoft.CodeAnalysis.CSharp, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
+- 'SonarAnalyzer.ShimLayer, Version=9.32.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
 - 'System.Collections.Immutable, Version=1.1.37.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'
-# Number of references: 4
+# Number of references: 5
 
 ---
-Assembly: 'SonarAnalyzer.CSharp, Version=9.30.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+Assembly: 'SonarAnalyzer.CSharp, Version=9.32.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
 Relative path: 'SonarAnalyzer.CSharp.dll'
 
 Referenced assemblies:
 - 'Microsoft.CodeAnalysis, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'Microsoft.CodeAnalysis.CSharp, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'Microsoft.CodeAnalysis.Workspaces, Version=1.3.1.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'
 - 'netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51'
-- 'SonarAnalyzer, Version=9.30.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
-- 'SonarAnalyzer.CFG, Version=9.30.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+- 'SonarAnalyzer, Version=9.32.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+- 'SonarAnalyzer.CFG, Version=9.32.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
+- 'SonarAnalyzer.ShimLayer, Version=9.32.0.0, Culture=neutral, PublicKeyToken=c5b62af9de6d7244'
```

Comment:
we might need to add this assembly in the vsix manifest https://github.com/SonarSource/sonarlint-visualstudio/blob/63919518a1f9fb743f9ed56db3cb0e1e423b4d30/src/Integration.Vsix/Manifests/VS2022/source.extension.vsixmanifest#L37

Reply:
Probably would be safer to do that regardless, but you can test that by looking at the loaded assemblies in the debugger